ci/Jenkinsfiles/build.groovy

```diff
@@ -885,6 +886,16 @@ pipeline {
       script {
         if (!isPullRequest() && env.DRY_RUN != "true") {
           currentBuild.description = "Build ${VERSION}"
+          if(!hudson.model.Result.FAILURE.equals(currentBuild.getPreviousBuild()?.getResult())) {
+            slackSend(channel: "${SLACK_CHANNEL}", message: "Successfully built ${BRANCH_NAME}/${BUILD_NUMBER}: ${BUILD_URL}")
```

Comment:
Could color be added?

Reply:
Added color to all messages
good for success
danger for failure
and #0167FF for promotion start notice